lib/dicon/secure_shell.ex

```diff
@@ -48,8 +53,8 @@ defmodule Dicon.SecureShell do
       :ok -> :ok
       {:error, {:already_started, :ssh}} -> :ok
       {:error, reason} ->
-        Mix.raise "Could not start ssh application: " <>
-          Application.format_error(reason)
+        {:error, "could not start ssh application: " <>
```

Comment:
With this error reason we have nested colons in the final error message:
```
** (Mix.Error) Error in executor Dicon.SecureShell: could not start ssh application: already started application ssh
```
Also `Error` looks redundant.
Maybe use something like `** (Mix.Error) [Dicon.SecureShell] could not ...`?